packages/ctags/build.sh

```diff
@@ -2,22 +2,15 @@ TERMUX_PKG_HOMEPAGE=https://ctags.io/
 TERMUX_PKG_DESCRIPTION="Universal ctags: Source code index builder"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="2:6.1.0"
-TERMUX_PKG_SRCURL=https://github.com/universal-ctags/ctags/archive/refs/tags/v${TERMUX_PKG_VERSION:2}.tar.gz
+TERMUX_PKG_VERSION="6.1.0"
+TERMUX_PKG_REVISION=2
```

Comment:
Bump to `TERMUX_PKG_REVISION=1` is enough
The 2 in `2:6.1.0` is epoch 2 and should be left as untouched.

Reply:
Ok, I simply removed revision then.

Reply:
No. I mean leave version untouched and set revision to 1. Refer CI failure message.

Reply:
Ok, put it back.